### PR TITLE
Feat: Button Component

### PR DESCRIPTION
**Pull request by alain-kubwayo**
## Describe your changes
Added two versions of the Button component, both default and large versions, with animated arrow.

## Issue ticket number and link
- ID: 006
- Link:  [Notion](https://www.notion.so/apeunit/Button-Component-7c664707f7ba4930b1375fc30930632b) 

## Tasks completed
- [x] Added default button component
- [x] Added large button component
 
## Tasks not completed
- None

## Screenshots (if needed)
button-preview

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
+import {DefaultButton, LargeButton} from "../components/ui/Button";
+
 export default function Home() {
   return (
     <>
       <h1 className="text-center font-bold text-3xl text-indigo-700">Unit A landing Page</h1>;
+      <DefaultButton text="Learn more" hasArrow={true} />
+
+      <div className="my-10"></div>
```
Comment (Ntwali-Josue) [inline pages/index.js:9]: Avoid using empty tags

**Review by Ntwali-Josue — APPROVED** [review]
Good work @alain-kubwayo and @Delyc, looks good to me. I have added a comment about not using empty tags, but it can be removed on the next PR.

File: pages/index.js (hunk 1 above)
Comment (Delyc) [reply pages/index.js]: Done

**Review by mhenriette — APPROVED** [review]
Looks great!!

**Pull request merged**